src/plugins/home/server/services/sample_data/data_sets/logs/field_mappings.ts

```diff
@@ -102,6 +102,9 @@ export const fieldMappings = {
   bytes: {
     type: 'long',
   },
+  bytes_histogram: {
+    type: 'histogram',
+  },
```

Comment:
For reviewers: You can test the histogram data in the sample logs, but I think this is probably not a good idea to merge. So I'll plan to remove these changes before merging.